CDCgov/cdc-open-viz
# Enhancement [DEV-9652] use full legend container for gradient legend

## [DEV-9652](https://websupport-cdc.msappproxy.net/browse/DEV-9652)

use full legend container width for gradient legend

## Testing Steps

1. Create or view map or chart with gradient legend
2. Observe or inspect legend container and confirm it is filled by legend

## Self Review

- I have added testing steps for reviewers
- I have commented my code, particularly in hard-to-understand areas
- My changes generate no new warnings
- New and existing unit tests are passing

## Screenshots (if applicable)

<img width="423" alt="Screenshot 2024-10-29 at 6 02 38 PM" src="https://github.com/user-attachments/assets/245c4887-c0dd-46d5-a476-f4421bf00869">
